sdk/storage/platform-matrix-all-versions.json

```diff
@@ -0,0 +1,20 @@
+{
+  "matrix": {
+    "Agent": {
+      "ubuntu-18.04": { "OSVmImage": "MMSUbuntu18.04", "Pool": "azsdk-pool-mms-ubuntu-1804-general" },
+      "windows-2019": { "OSVmImage": "MMS2019", "Pool": "azsdk-pool-mms-win-2019-general" }
+    },
+    "JavaTestVersion": [ "1.8", "1.11" ],
+    "AZURE_TEST_HTTP_CLIENTS": [ "okhttp", "netty" ],
+    "TestGoals": "surefire:test",
+    "AZURE_LIVE_TEST_SERVICE_VERSIONS": [
```

Comment:
`AZURE_LIVE_TEST_SERVICE_VERSIONS` should this be `AZURE_LIVE_TEST_SERVICE_VERSION` - I'm assuming each job gets just one value from this list ?

Reply:
The values are passed singular right now, but if you want to support a csv of versions, that could be an option hence the name. This is what it's named in net so was trying to be consistent, but can change if you don't want to support multiple versions.

Reply:
in .NET we learnt that testing more than 1 service version per job isn't great - leads to longer run time, higher chance that job will fail and makes "retry failed jobs" less effective as scope of one job grows.
@alzimmermsft I suggest to make assumption that one SV runs in one pipeline job for JUnit tests as well.